generators/aws-containers/prompts.js

```diff
@@ -200,32 +222,31 @@ function askPerformances() {
 function promptPerformance(config, awsConfig = { performance: 'low' }) {
     if (this.abort) return null;
 
-    const performanceLevels = _.keys(PERF_TO_CONFIG)
+    const prodDatabaseType = config.prodDatabaseType;
+
+    if(prodDatabaseType === databaseTypes.postgresql) {
+        this.log.ok(` ⚠️ Postgresql databases are currently only supported by Aurora on high-performance database instances`);
```

Comment:
This is not ok, The hot tub is too hot.
change the log and use chalk to either use orange or red

Reply:
Dead to me